lib/fog/profitbricks/requests/compute/get_attached_cdrom.rb

```diff
@@ -48,6 +48,8 @@ def get_attached_cdrom(datacenter_id, server_id, cdrom_image_id)
               :method  => 'GET',
               :path    => "/datacenters/#{datacenter_id}/servers/#{server_id}/cdroms/#{cdrom_image_id}?depth=1"
           )
+        rescue => error
+          Fog::Errors::NotFound.new(error)
```

Comment:
Same thing here. What does this api return in case of not having any attached CD ROM?